apps/communications/contacts/test/marionette/search_test.js

```diff
@@ -16,37 +16,44 @@ marionette('Contacts > Search', function() {
   });
 
   suite('Search Mode', function() {
-    test.skip('Can enter and exit search mode', function() {
-
+    test('Can enter and exit search mode', function() {
       subject.addContact();
 
-      client.helper.waitForElement(selectors.searchLabel)
-        .click();
-
-      client.helper.waitForElement(selectors.searchInput)
-        .sendKeys('testme');
-
-      client.helper.waitForElement(selectors.searchCancel)
-        .click();
+      client.helper.waitForElement(selectors.searchLabel).click();
+      var result = client.helper.waitForElement(selectors.searchResultFirst);
+      assert.ok(result.displayed(), 'Search view was not opened');
 
+      client.helper.waitForElement(selectors.searchCancel).click();
       var listView = client.helper.waitForElement(selectors.list);
-      assert.ok(listView.displayed(), 'List view is shown.');
+      assert.ok(listView.displayed(), 'List view was not shown');
     });
 
-    test.skip('Search text is displayed correctly', function() {
+    test('Search text is displayed correctly', function() {
       var details = details || {
         givenName: 'A%%&'
       };
 
       subject.addContact(details);
       client.helper.waitForElement(selectors.searchLabel).click();
-
       client.helper.waitForElement(selectors.searchInput).sendKeys('a');
       client.helper.waitForElement(selectors.searchResultFirst).text(
         function(error, textContent) {
           assert.equal('A%%&', textContent);
       });
     });
-  });
 
+    test('Search gets updated if contact changes', function() {
+      subject.addContact();
```

Comment:
probably we can use contacts.createMozContact() as a safer way to create the initial contact data needed for this test